### ISSUE TITLE
imports paths and redirects

### ISSUE THREAD

**Issue by markbrown4** [ModuleLoader/browser-es-module-loader#8]
I noticed a difference between the native loading and this polyfill when importing from a route that redirects.

`import _ from 'https://unpkg.com/lodash-es'`
https://lodash-in-the-browser-vwtxyusbpk.now.sh

That results in 404's for some import paths.

When I link to the module directly everything resolves correctly.

`import _ from 'https://unpkg.com/lodash-es@4.17.7/lodash.js'`
https://lodash-in-the-browser-vxldnvahwg.now.sh

**Comment by guybedford** [comment]
Yes this is a bug between the implementations. For SystemJS this is being tracked here - https://github.com/systemjs/systemjs/issues/1754#issuecomment-337174682.
